openedx/core/djangoapps/profile_images/images.py

```diff
@@ -0,0 +1,128 @@
+"""
+Image file manipulation functions related to profile images.
+"""
+from cStringIO import StringIO
+
+from django.conf import settings
+from django.core.files.base import ContentFile
+from PIL import Image
+
+from ..user_api.accounts.api import get_profile_image_storage
+
+
+class DevMsg(object):
+    """
+    Holder for pseudo-constants.
+    """
+    FILE_TOO_LARGE = 'Maximum file size exceeded.'
+    FILE_TOO_SMALL = 'Minimum file size not met.'
+    FILE_BAD_TYPE = 'Unsupported file type.'
+    FILE_BAD_EXT = 'File extension does not match data.'
+    FILE_BAD_MIMETYPE = 'Content-Type header does not match data.'
+
+
+class ImageFileRejected(Exception):
```

Comment:
This seems like an oddly named Exception. Usually the name of the exception describes what failed (ValueError, ValidationError, etc.). This one doesn't say what is wrong, but states what action was taken.

Is it necessary to declare our own exception type? If that is considered good practice, I would suggest something like  ImageValidationError,